Dispatchfile

```diff
@@ -4,7 +4,7 @@
 
 
 load("github.com/mesosphere/dispatch-catalog/starlark/stable/git@0.0.7", "git_resource")
-#TODO @faiq: once 0.0.5 of this package is released switch to that ref
+# TODO @faiq: once 0.0.5 of this package is released switch to that ref
```

Comment:
what... was i talking about

Reply:
probably was below 0.0.5, then later on we updated to 0.0.7 and left your comment there